tensorboard/plugins/profile/overview_page/overview-page.html

```diff
@@ -105,6 +105,8 @@
         <paper-card heading="Performance Summary">
           <div class="card-content">
             <p><b>Average step time</b> (lower is better): <b><span class="steptime-average" >[[_steptime_ms_average]] ms</span> </b> <i style="opacity:0.7">(standard deviation = <span>[[_steptime_ms_stddev]]</span> ms)</i></p>
+            <p>- Average infeed: <span>[[_infeed_ms_average]]</span> ms</p>
```

Comment:
if you are going to prefix it with `-`, it might be better to use more semantically correct tag, `<ul>` and `<li>`

Reply:
Done. Changed to <ul> and <li>.